src/lib/Toast/properties.ts

```diff
@@ -0,0 +1,23 @@
+export type ToastType = 'success' | 'error' | 'info' | 'warn';
+
+export type ToastProperties = {
+  type: ToastType;
```

Comment:
lets make type an optional param. users can avoid passing type in case they want to show some kind of custom message.

Reply:
updated.